plugins/com.google.cloud.tools.eclipse.appengine.validation.test/src/com/google/cloud/tools/eclipse/appengine/validation/XmlSourceValidatorTest.java

```diff
@@ -173,4 +180,21 @@ public void testGetProject() throws CoreException {
     assertEquals(project, testProject);
   }
 
+  @Test
+  public void testNoNullPointerExceptionOnNonFacetedProject()
```

Comment:
This method name is a bit too specific to the bug!  Maybe `testIgnoreNonFacetedProject()`?

Reply:
I'll name it as `testNoErrorOnNonFacetedProject()`, because the test doesn't actually check if it ignores and does not validate XMLs in a non-faceted project. (It just checks if it will crash.